yunwoo/baekjoon/수학/10250-ACM-호텔.js

```diff
@@ -0,0 +1,32 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const t = +input[0];
+  let result = "";
+  for (let i = 1; i <= t; i++) {
+    const [h, w, n] = input[i].split(" ").map(Number);
+    const floor = n % h === 0 ? h : n % h;
+    const roomNumber = Number.isInteger(n / h) ? n / h : Math.ceil(n / h);
+
+    result +=
+      roomNumber < 10 ? `${floor}0${roomNumber}\n` : `${floor}${roomNumber}\n`;
+  }
+  console.log(result);
```

Comment:
```suggestion
  const t = +input[0];
  let result = "";
  
  for (let i = 1; i <= t; i++) {
    const [h, w, n] = input[i].split(" ").map(Number);
    const floor = n % h === 0 ? h : n % h;
    const roomNumber = Math.ceil(n / h);
    
    result +=
      roomNumber < 10 ? `${floor}0${roomNumber}\n` : `${floor}${roomNumber}\n`;
  }
  console.log(result);
```
- 풀이 잘봤습니다 🙇‍♂️
- roomNumber를 isInterger를 통해 정수인지 굳이 판별 하실 필요 없습니다. Math.ceil은 올림해주기 때문에 정수일 경우에는 그냥 값 그대로를 가집니다. ex) 1.5 -> 2 / 1 -> 1